spec/controllers/queue_items_controller_spec.rb

```diff
@@ -81,32 +77,117 @@
     end
   end
 
-  describe "DELETE destory" do 
-    it "should redirect to my queue page" do
-      session[:user_id] = Fabricate(:user).id
-      queue_item = Fabricate(:queue_item)
-      delete :destroy, id: queue_item.id
-      expect(response).to redirect_to my_queue_path
+  describe "DELETE destory" do
+    context "with authenticated user" do
+
+      let(:thupten) { Fabricate(:user)}
+
+      before { session[:user_id] = thupten.id }
+
+      it "should redirect to my queue page" do
```

Comment:
I would recommend to avoid using word "should" in examples message - it does not add any value, rather use 3rd person singular for verb - Read more [here](http://www.betterspecs.org/#should)